src/components/tables/DataTable.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="dataTable">
+    <table ref="table" class="data-table table-responsive d-print-inline" style="width:100%">
+      <thead>
+        <tr>
+          <th v-for="(col, colIndex) in columns" :key="col">
+            {{ col.name }}
+            <Tooltip @click.stop :content="col.help"></Tooltip>
+            <input @click.stop @input="filter($event, colIndex)" data type="text" class="filter" />
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="row in rows" :key="row">
+          <td v-for="value in row" :key="value">
+            <DataTableCell :cellValue="value"></DataTableCell>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</template>
+
+<script>
+import $ from 'jquery';
```

Comment:
as jQuery is already available in our system in the window, why do you need to import it from a library?

Reply:
Removed import from the library. Now using the global jQuery object.